.readthedocs.yml

```diff
@@ -1,24 +1,20 @@
 # .readthedocs.yml
 # Read the Docs configuration file
-# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 
-# Required
 version: 2
 
 build:
-  os: ubuntu-22.04
+  os: "ubuntu-22.04"
   tools:
     python: "3.8"
+  jobs:
+    post_create_environment:
+      # Install poetry
+      - pip install poetry
+    post_install:
+      # Install dependencies
+      - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH poetry install --only docs
```

Comment:
This is an env variable? What should it be set to?

Reply:
$READTHEDOCS_VIRTUALENV_PATH is a predefined env variable set by Read the Docs in the build environment. It's the path to the virtual env used by 'Read the Docs'  when building the project's documentation.

Reply:
https://docs.readthedocs.io/en/stable/build-customization.html